apps/video/js/db.js

```diff
@@ -153,6 +153,7 @@ function addVideo(videodata) {
 
   // Create the thumbnail view for this video and insert it at the right spot
   var view = thumbnailList.addItem(videodata);
+
```

Comment:
You should probably remove this new line before landing. I assume this is an accidental insertion from adding and removing logging statements or something.